interfaces/builtin/location_control_test.go

```diff
@@ -87,7 +87,9 @@ func (s *LocationControlInterfaceSuite) TestConnectedPlugSnippetUsesSlotLabelAll
 		Name:      "location",
 		Interface: "location",
 		Apps:      map[string]*snap.AppInfo{"app1": app1, "app2": app2},
-	}, nil)
+	}, nil,
+
```

Comment:
Spacing is odd here. Can you remove the newline and see if this looks more like before please?

Reply:
Fixed. Indeed, it came from gofmt...